services/auth/src/api/routes/profile.py

```diff
@@ -0,0 +1,73 @@
+from fastapi import APIRouter, Depends
+from sqlalchemy import select
+from sqlalchemy.ext.asyncio import AsyncSession
+
+from api.deps import AccessRequire, Session
+from models import User, Profile
+from schemas import MeResponse, ProfileSchema, ProfileUpdate
+
+router = APIRouter(prefix="/me", tags=["Profile"])
+
+
+@router.get("", response_model=MeResponse)
+async def get_me(
+    require: AccessRequire,
+    session: Session,
+) -> MeResponse:
+    """Get current user with profile."""
+    user_id = require.payload["sub"]
+    
+    # Load user with profile
+    result = await session.execute(
+        select(User).where(User.id == user_id)
+    )
+    user = result.scalar_one_or_none()
+    
+    if not user:
+        from fastapi import HTTPException, status
+        raise HTTPException(
+            status_code=status.HTTP_404_NOT_FOUND,
+            detail="User not found",
+        )
```

Comment:
Лучше вынести где все ексепшены в `core.exceptions`, создать там по примеру остальных ексепшенов новый `UserNotFound`, а импорт вынести наверх файла

```
from core.exceptions import UserNotFound
```
```suggestion
        raise UserNotFound()
```